File: VisualBasicAssignment/services/Alertdebug.vb
Imports System
Imports System.Windows.Forms

Namespace CustomAlertBoxDemo
    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Public Sub Alert(ByVal msg As String, ByVal type As Form_Alert.enmType)
            Dim frm = New Form_Alert()
            frm.showAlert(msg, type)
        End Sub

        Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs)
            Alert("Success Alert", Form_Alert.enmType.Success)
        End Sub

        Private Sub button2_Click(ByVal sender As Object, ByVal e As EventArgs)
            Alert("Warning Alert", Form_Alert.enmType.Warning)
        End Sub

        Private Sub button3_Click(ByVal sender As Object, ByVal e As EventArgs)
            Alert("Error Alert", Form_Alert.enmType.Error)
        End Sub

        Private Sub button4_Click(ByVal sender As Object, ByVal e As EventArgs)
            Alert("Info Alert", Form_Alert.enmType.Info)
        End Sub
    End Class
End Namespace

File: VisualBasicAssignment/services/Alertdebug.cs
using System;
using System.Windows.Forms;

namespace VisualBasicAssignment.CustomAlertBoxDemo
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        public void Alert(string msg, Form_Alert.enmType type)
        {
            var frm = new Form_Alert();
            frm.showAlert(msg, type);
            frm.Show();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            Alert("Success Alert", Form_Alert.enmType.Success);
        }

        private void button2_Click(object sender, EventArgs e)
        {
            Alert("Warning Alert", Form_Alert.enmType.Warning);
        }

        private void button3_Click(object sender, EventArgs e)
        {
            Alert("Error Alert", Form_Alert.enmType.Error);
        }

        private void button4_Click(object sender, EventArgs e)
        {
            Alert("Info Alert", Form_Alert.enmType.Info);
        }
    }
}